plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/BuildProgressSplitterPanelManager.kt

```diff
@@ -215,22 +217,38 @@ class BuildProgressSplitterPanelManager(private val project: Project) :
             }
 
             TransformationStatus.PREPARING -> {
-                loadingPanelText = message("codemodernizer.toolwindow.scan_in_progress.building", jdkVersion.description)
+                loadingPanelText = if (transformType == CodeTransformType.SQL_CONVERSION) {
+                    message("codemodernizer.toolwindow.scan_in_progress.transforming")
+                } else {
+                    message("codemodernizer.toolwindow.scan_in_progress.building", jdk.description)
+                }
                 statuses.update(BuildStepStatus.DONE, ProgressStepId.UPLOADING)
             }
 
             TransformationStatus.PREPARED -> {
-                loadingPanelText = message("codemodernizer.toolwindow.scan_in_progress.building", jdkVersion.description)
+                loadingPanelText = if (transformType == CodeTransformType.SQL_CONVERSION) {
+                    message("codemodernizer.toolwindow.scan_in_progress.transforming")
+                } else {
+                    message("codemodernizer.toolwindow.scan_in_progress.building", jdk.description)
+                }
                 statuses.update(BuildStepStatus.DONE, ProgressStepId.BUILDING)
             }
 
             TransformationStatus.PLANNING -> {
-                loadingPanelText = message("codemodernizer.toolwindow.scan_in_progress.planning")
+                loadingPanelText = if (transformType == CodeTransformType.SQL_CONVERSION) {
+                    message("codemodernizer.toolwindow.scan_in_progress.transforming")
```

Comment:
Might need to refer back to the UX, but do we skip Preparing / Prepared / Planning / Planned stages for SQL conversions or are we just displaying it as transforming?

Reply:
Technically in our backend the job will progress through those statuses since its the same state machine being used, but we don't show that to the user. Just straight from uploading -> transforming since we don't build and we don't create a plan for now